### ISSUE TITLE
Infer strong and/or emphasis in ombpdf

### ISSUE THREAD

**Issue by toolness** [18F/omb-eregs#783]
(This issue has been migrated from https://github.com/18F/omb-pdf/issues/25.)

A number of the policies use bold, italicized, and/or underlined text, e.g. page 2 of M-15-17:

> ![underline](https://user-images.githubusercontent.com/124687/33612356-40872d3a-d99f-11e7-9e62-2077aa1c0576.png)

It's straightforward enough to just convert italics to `<em>` and bold to `<strong>` but I'm not sure what to do about underlined text.  In the above screenshot, it seems like it's being used as `<strong>` but I'm not sure.

**Comment by carodew** [comment]
I think there's a strong argument to be made that we don't underline any text that is not a link bc a) it's a stylistic rendering, and doesn't change the content (as long as we render the visual emphasis in an equivalent way) and b) it can make text look like a link, especially for colorblind users.

**Comment by toolness** [comment]
I agree! I wonder if we could somehow present this to users who import PDFs as a kind of question: "hey, this part of the PDF was underlined, but only links in the HTML version should be underlined, so do you want to convert this to bold or italic?"

**Comment by tadhg-ohiggins** [comment]
At some point last year we discussed emphasized text at the start of a list item and I seem to recall we decided that it was really the title of the list item, so here we wouldn't want to preserve underline or even emphasis for `Improve Educational Outcomes and Life Outcomes for native Youth.`, but rather recognize that it's the title of the list item. (Lots of these documents seem to use the titled-list-item approach.)

As for the second underline, `Bureau of Indian Education`, I have no idea what the intent is there, since I don't understand why you would underline the term, particularly when you didn't underline `Department of Education` in the same sentence.

I don't think we should give importing users the option of retaining the underlining, for the reasons @carodew mentioned.

So I think the ideal here is that the importer recognizes the opening list item title, emphasizes `Bureau of Indian Education` (because traditionally I think underlining is closer to that than `strong`), and emphasizes the italicized section. I suspect that we'll only be able to do the first, and that the other two would have to be done in the editor (where the user would have emphasis/strong choices, but not underline, IMO).
